src/tokenizer/stemmer/en.ts

```diff
@@ -0,0 +1,189 @@
+const step2List = {
```

Comment:
Non blocking comment on this file.

If you can, it would be great to extract all constant RegExp expression at the module level and declare them as RegExp when possible so we might get a small to tiny performance improvement.